Sources/Concrete/Controller/SpeechController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  SpeechController.swift
```

Comment:
Can you add in the description how you would initialise and call this speech controller